data/json/items/comestibles.json

```diff
@@ -2620,8 +2620,8 @@
         "price" : 500,
         "material" : "flesh",
         "volume" : 1,
-        "charges" : 1,
-        "stack_size" : 2,
+        "stack_size" : 4,
```

Comment:
That would make it 2x as volume-efficient as cooking oil then.

While that makes sense from balancing purposes (lard is harder to acquire than cooking oil), this isn't realistic and as far as I recall, the entire reason for doubling the number of charges for cooking oil was realism.